Podcast/PrivacyPolicyViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  PrivacyPolicyViewController.swift
+//  Podcast
+//
+//  Created by Jack Thompson on 3/5/18.
+//  Copyright © 2018 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+class PrivacyPolicyViewController: UIViewController, UIWebViewDelegate {
+
+    var webView: UIWebView!
+    
+    var privacyPolicyURL = "http://www.cornellappdev.com/recast-privacy-policy"
```

Comment:
let's add this url to `Keys.plist` instead just like the `api-url` and commit `Keys.plist` 
BUT make sure to commit with the value of api-url being the empty string